app/Resources/views/events/show.html.twig

```diff
@@ -26,6 +26,71 @@
 
         {% include 'events/_metadata.html.twig' with {'event': event} %}
 
+        {% if filtersMissing %}
+        <section class="event-errorbox error">
+            <h3>{{ msg('error-filters-header') }}</h3>
+            <p>
+                {{ msg('error-filters-desc') }}
+                <a href="https://meta.wikimedia.org/wiki/Special:MyLanguage/Event_Metrics#adding-participants"
+                   title="{{ msg('error-filters-desc-tooltip') }}">
+                    <span class="glyphicon glyphicon glyphicon-question-sign"></span>
+                </a>
+            </p>
+            <ul>
+                <li>{{ msg('error-filters-participants-or-categories') }}</li>
+                <li>{{ msg('error-filters-every-wiki') }}</li>
+                <li>{{ msg('error-filters-participants-desc') }}</li>
+                <li>{{ msg('error-filters-categories-desc') }}</li>
+                <li>{{ msg('error-filters-combination') }}</li>
+            </ul>
+        </section>
+
+        {% elseif event.numParticipants == 0 and wikisWithoutCats %}
+        <section class="event-errorbox warn">
+            <h3>{{ msg('category-warning-header') }}</h3>
+            <p>
+                {{ msg('category-warning-desc') }}
+                <a href="https://meta.wikimedia.org/wiki/Special:MyLanguage/Event_Metrics#adding-categories"
+                   title="{{ msg('category-warning-desc-tooltip') }}">
+                    <span class="glyphicon glyphicon glyphicon-question-sign"></span>
+                </a>
+            </p>
+            <ul>
+                <li>
+                    {{ msg('error-filters-categories-missing') }}
+                    <strong>
+                        {% for noCatWiki in wikisWithoutCats %}
+                            {{ noCatWiki.domain }}{% if not loop.last %},{% endif %}
+                        {% endfor %}
+                    </strong>
+                </li>
+                <li>{{ msg('error-filters-every-wiki') }}</li>
+                {% if event.wikiByDomain('www.wikidata') %}
+                <li>{{ msg('error-filters-wikidata') }}</li>
+                {% endif %}
+                <li>{{ msg('error-filters-participants-desc') }}</li>
+                <li>{{ msg('error-filters-categories-desc') }}</li>
+            </ul>
+        </section>
+
+        {% else %}
+            <div class="alert alert-success">
+                {% if event.startUTC > date() %}
+                    {{ msg('event-in-future') }}
+                    (<a href="{{ path('EditEvent', {'programId': program.id, 'eventId': event.id}) }}">{{ msg('settings')|lower }}</a>)
+                {% else %}
+                    {{ msg('event-no-data') }}
+                    <a href="#" class="event-process-link">{{ msg('calculate-totals') }}</a>.
```

Comment:
After I click "Calculate totals" the message does not disappear (which it did before).

Reply:
Fixed. Now checks for job (as it did before).